json-endpoints/class.wpcom-json-api-list-post-types-endpoint.php

```diff
@@ -9,6 +9,8 @@ class WPCOM_JSON_API_List_Post_Types_Endpoint extends WPCOM_JSON_API_Endpoint {
 		'map_meta_cap' => 'map_meta_cap',
 		'cap'          => 'capabilities',
 		'hierarchical' => 'hierarchical',
+		'show_ui'   => 'show_ui',
```

Comment:
Very minor, but maybe we can add space between the key and value here so that `=>` aligns with the ones above it.